podcasts/PlaybackItem.swift

```diff
@@ -15,9 +15,17 @@ class PlaybackItem: NSObject {
 
     func createPlayerItem() -> AVPlayerItem? {
         guard let url = EpisodeManager.urlForEpisode(episode) else { return nil }
-
-        let customHeaders = [ServerConstants.HttpHeaders.userAgent: ServerConstants.Values.appUserAgent]
-        let asset = AVURLAsset(url: url, options: ["AVURLAssetHTTPHeaderFieldsKey": customHeaders])
+        var options = [String : Any]()
```

Comment:
There's a warning here:

> Colon Spacing Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. (colon)

```suggestion
        var options = [String : Any]()
```

Reply:
@johnspurlock-skymethod can you please commit this suggestion? Then we'd be able to merge this PR :)